azure-pipelines.yml

```diff
@@ -20,6 +20,12 @@ pr:
   paths:
     exclude:
       - .github
+      - .azure-pipelines
+    include:
+      - .azure-pipelines/azure-pipelines-build.yml
+      - .azure-pipelines/azure-pipelines-image-template.yml
+      - .azure-pipelines/azure-pipelines-job-groups.yml
+      - .azure-pipelines/run-test-template.yml
```

Comment:
The PR will be blocked, if someone only changes one/some of the files, suggest not add it.